### ISSUE TITLE
Training loss is more erratic than eval loss

### ISSUE THREAD

**Issue by RustyPanda** [RustyPanda/zoobot#22]
<img width="1146" alt="screen shot 2018-06-26 at 13 16 11" src="https://user-images.githubusercontent.com/7740526/41910928-22042eb4-7943-11e8-9e9f-ccf78cde5180.png">

Red - training loss.

There's no good reason training loss should be so highly varied. As this is the key optimisation param, this might be very significant for training.

This may be related to the magic numbers in the stratified sample plots (#21) : 32 and 12800.

This may be related to the strange tensorboard keyerror.

Working theory: batch size with stratified sampling is variable and often small, one or a handful of images.
